translator/sample files/cotrip_data/cotrip_1.json

```diff
@@ -0,0 +1,108 @@
+{

```

Comment:
Is this file used?

Reply:
we are using it to run the program

Reply:
It's never mentioned anywhere. If we have multiple test input files, let's update the documentation to reflect this.

Reply:
### Execution for COtrip translator

#### Step 1: Run the translator script (from Work_Zone/translator/source code)

```
python cotrip_translator.py -i inputfile.json -o outputfile.geojson
```
Example usage:
```
python cotrip_translator.py -i '../sample files/cotrip_data/cotrip_1.json' 



i used that file name in documentation..in fact i deleted the cotrip_1.json from that directory.

Reply:
Ah! I was thinking of `cotrip_2.json` below. I do see the reference to `cotrip_1.json` though.